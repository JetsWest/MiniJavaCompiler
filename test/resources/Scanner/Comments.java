a = 4;
// Ignore this!
b = 5;