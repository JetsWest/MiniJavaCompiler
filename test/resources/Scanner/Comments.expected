ID(a) BECOMES DIGIT(4) SEMICOLON ID(b) BECOMES DIGIT(5) SEMICOLON